src/tests/ftest/container/container_redundancy.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
```

Comment:
I think this test should go in the rebuild folder instead of container

Reply:
Have updated test scripts to rebuild.